docs/getting_started.rst

```diff
@@ -25,14 +25,7 @@ file, place it in the ``cfme_tests`` repository (along ``conftest.py``):
 
   #!/usr/bin/env bash
 
-  if hash dnf;
-  then
-    YUM=dnf
-  else
-    YUM=yum
-  fi
-
-  sudo $YUM install -y python-virtualenv gcc postgresql-devel libxml2-devel libxslt-devel zeromq3-devel libcurl-devel redhat-rpm-config gcc-c++ openssl-devel libffi-devel
+  pkcon install -y python-virtualenv gcc postgresql-devel libxml2-devel libxslt-devel zeromq3-devel libcurl-devel redhat-rpm-config gcc-c++ openssl-devel libffi-devel
```

Comment:
As Pete said, this is a solution for the issue of different package managers in different distros... It doesn't provide us with cross-distro support because of the differing package names however. I'm not sure we should be spending time on this btw, seeing as it will be replaced by the containerized setup rather soon.